<?xml version="1.0" encoding="UTF-8"?>
<testrun name="EventDetectionIT" project="EnterpriseEventProcessing" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="com.marisoft.ziba.cep.tests.jms.EventDetectionIT" time="1.988">
    <testcase name="testsubmitEventAndDetectionOfIt" classname="com.marisoft.ziba.cep.tests.jms.EventDetectionIT" time="1.988">
      <error>org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.apache.activemq.xbean.XBeanBrokerService#0': Invocation of init method failed; nested exception is java.io.IOException: Failed to bind to server socket: tcp://localhost:13208 due to: java.net.BindException: Address already in use
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:136)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:407)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1545)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:539)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:475)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:304)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:228)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:300)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:195)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:700)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:760)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:482)
	at org.springframework.context.support.ClassPathXmlApplicationContext.&lt;init&gt;(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.&lt;init&gt;(ClassPathXmlApplicationContext.java:83)
	at com.espertech.esperio.SpringContextLoader.createSpringApplicationContext(SpringContextLoader.java:110)
	at com.espertech.esperio.SpringContextLoader.init(SpringContextLoader.java:76)
	at com.espertech.esper.core.service.EPServiceProviderImpl.loadAdapters(EPServiceProviderImpl.java:728)
	at com.espertech.esper.core.service.EPServiceProviderImpl.doInitialize(EPServiceProviderImpl.java:635)
	at com.espertech.esper.core.service.EPServiceProviderImpl.&lt;init&gt;(EPServiceProviderImpl.java:89)
	at com.espertech.esper.client.EPServiceProviderManager.getProviderInternal(EPServiceProviderManager.java:117)
	at com.espertech.esper.client.EPServiceProviderManager.getProvider(EPServiceProviderManager.java:88)
	at com.marisoft.ziba.cep.jms.EsperEventProcessingAgent.start(EsperEventProcessingAgent.java:33)
	at com.marisoft.ziba.cep.tests.jms.EventDetectionIT.testsubmitEventAndDetectionOfIt(EventDetectionIT.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
Caused by: java.io.IOException: Failed to bind to server socket: tcp://localhost:13208 due to: java.net.BindException: Address already in use
	at org.apache.activemq.util.IOExceptionSupport.create(IOExceptionSupport.java:33)
	at org.apache.activemq.transport.tcp.TcpTransportServer.bind(TcpTransportServer.java:138)
	at org.apache.activemq.transport.tcp.TcpTransportFactory.doBind(TcpTransportFactory.java:60)
	at org.apache.activemq.transport.TransportFactory.bind(TransportFactory.java:124)
	at org.apache.activemq.broker.TransportConnector.createTransportServer(TransportConnector.java:310)
	at org.apache.activemq.broker.TransportConnector.getServer(TransportConnector.java:136)
	at org.apache.activemq.broker.TransportConnector.start(TransportConnector.java:201)
	at org.apache.activemq.broker.BrokerService.startTransportConnector(BrokerService.java:2471)
	at org.apache.activemq.broker.BrokerService.startAllConnectors(BrokerService.java:2385)
	at org.apache.activemq.broker.BrokerService.doStartBroker(BrokerService.java:684)
	at org.apache.activemq.broker.BrokerService.startBroker(BrokerService.java:642)
	at org.apache.activemq.broker.BrokerService.start(BrokerService.java:578)
	at org.apache.activemq.broker.BrokerService.autoStart(BrokerService.java:539)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleElement.invoke(InitDestroyAnnotationBeanPostProcessor.java:349)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:300)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:133)
	... 46 more
Caused by: java.net.BindException: Address already in use
	at java.net.PlainSocketImpl.socketBind(Native Method)
	at java.net.PlainSocketImpl.bind(PlainSocketImpl.java:383)
	at java.net.ServerSocket.bind(ServerSocket.java:328)
	at java.net.ServerSocket.&lt;init&gt;(ServerSocket.java:194)
	at javax.net.DefaultServerSocketFactory.createServerSocket(ServerSocketFactory.java:170)
	at org.apache.activemq.transport.tcp.TcpTransportServer.bind(TcpTransportServer.java:134)
	... 64 more

</error>
    </testcase>
  </testsuite>
</testrun>
